dnf-plugins-core.spec

```diff
@@ -69,6 +70,8 @@ Group:      System Environment/Base
 BuildRequires:  python3-devel
 BuildRequires:  python3-dnf >= %{dnf_lowest_compatible}
 BuildRequires:  python3-dnf < %{dnf_not_compatible}
+# not needed b/c we're using unittest.mock
```

Comment:
Hmm, I've added these b/c nobody will see the commit message after a while but they will see the comments in the spec file. The idea here is to not add python3-mock as unnecessary BuildRequires. 

I can remove them if you still feel that way.